assignments/arrays.js

```diff
@@ -62,18 +313,44 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // }
 
 // ==== Challenge 1 ====
-// The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// The dealer can't recall the information for a car with an id of 33 on his lot. 
+// Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
+const getCar = []
+for (let i = 0; i < inventory.length; i++) {
+    if (inventory[i].id === 33) {
+        // getCar.push(inventory[i])
+        console.log(`car ${inventory[i].id} is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model}`)
+    }
+}
+console.log(getCar);
+
+
+console.log(inventory[32]);
+
+
+
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+
+let lastCar = inventory[49];
+inventory.lastIndexOf(lastCar)
+console.log(lastCar);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+
+const sortedAlphabetically = [];
+for (let i = 0; i === inventory.car_model; i++) {
```

Comment:
```suggestion
for (let i = 0; i < inventory.length; i++) {
```

Reply:
the 2nd parameter of the for loop is the "condition" to be evaluated before each loop iteration. It must evaluate to 'true' for the loop to continue, or in this case- to begin.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for